crates/vm/Cargo.toml

```diff
@@ -35,6 +35,7 @@ eyre.workspace = true
 derivative.workspace = true
 static_assertions.workspace = true
 getset.workspace = true
+dashmap = { version = "6.1.0", features = ["rayon"] }
```

Comment:
i'm not really sure, but put the `"dashmap/rayon"` feature in `"parallel"` below?